src/rgw/driver/rados/rgw_sal_rados.cc

```diff
@@ -1146,13 +1148,75 @@ int set_committed_logging_object(RadosStore* store,
   return rgw_rados_operate(dpp, io_ctx, obj_name_oid, std::move(op), y);
 }
 
+int get_logging_temp_object_pool (const DoutPrefixProvider *dpp,
```

Comment:
would it make sense to move this function to `rgw_bl_rados` ?

Reply:
It might be better to have this here as this is used only by the functions in this file.